src/tests/v1/test_moderation_events.py

```diff
@@ -0,0 +1,244 @@
+from django.api.models.moderation_event import ModerationEvent
```

Comment:
We couldn't used django module in integration tests

Reply:
Fixed! It was initially coderabbit suggestion and it was wrong.